packages/react-core/src/components/Tabs/Tabs.tsx

```diff
@@ -361,6 +364,7 @@ export class Tabs extends React.Component<TabsProps, TabsState> {
             showScrollButtons && !isVertical && styles.modifiers.scrollable,
             usePageInsets && styles.modifiers.pageInsets,
             !hasBorderBottom && styles.modifiers.noBorderBottom,
+            hasSecondaryBorderBottom && styles.modifiers.borderBottom,
```

Comment:
would this work to keep a single prop?

```suggestion
            !isSecondary && !hasBorderBottom && styles.modifiers.noBorderBottom,
             isSedoncary && hasBorderBottom && styles.modifiers.borderBottom,
```

Reply:
I *think* that would still cause breaking changes since `hasBorderBottom` defaults to true, so it would require consumers to add `hasBorderBottom={false}` if they want to keep their secondary tabs without a bottom border.